web/prql-codemirror-demo/README.md

```diff
@@ -0,0 +1,18 @@
+# CodeMirror with PRQL demo
+
+This is a demo of CodeMirror with PRQL. We don't have any published `lang-prql`
+or `prql-lezer` package yet.
+
+## Instructions
+
+Build `prql-lezer` then copy the files from `/grammars/prql-lezer/dist/` to
+`src/lang-prql/prql-lezer/`.
+
+```
+mkdir src/lang-prql/prql-lezer
+cd grammars/prql-lezer/
+npm run build
```

Comment:
So this `npm run build` doesn't work yet because there doesn't yet exist a `package.json` in `grammars/prql-lezer/`?

I think it would be best to merge this demo when the instructions work ... would you mind turning `prql-lezer` into a package? Note that it shouldn't be necessary to publish it on NPM just for the demo since I think the dependency could be specified with a [local path](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#local-paths).

Reply:
That `npm run buld` does work, because there does exist a `package.json` in `grammars/prql-lezer`.
https://github.com/PRQL/prql/blob/main/grammars/prql-lezer/package.json

I suppose I could fix the path in the instructions.

Reply:
Let's add the instructions to a job in `bulid-web.yaml` (either copy-paste or put in `Taskfile.yml`), and then they'll run in CI.

Alternatively we can publish them outside the repo — I don't want to slow down anyone from offering examples — fine to put them in a gist.

If we put them in the repo, though, then there's a higher bar for ensuring that the description is correct, and the only way I know to do that is to automate the tests...

Reply:
Do you want it in `build-web.yaml` or in the `Taskfile.yml`.

Maybe we should publish `prql-lezer` and `lang-prql` as two packages on npm then have the demo depend on `lang-prql` which in turn would depend on `prql-lezer`?

Reply:
It should run in `build-web.yaml`, and it can be nice to encapsulate it in `Taskfile.yml` and then call it from `build-web.yaml`, but it's a bit wore work, so not strictly necessary.

> Maybe we should publish `prql-lezer` and `lang-prql` as two packages on npm then have the demo depend on `lang-prql` which in turn would depend on `prql-lezer`?

I agree with publishing the grammar as a package! What's the advantage of having the two packages?

Reply:
OK, I would probably side with doing a single package at first — it's less work, and it's easy to split it up later than it is to combine it, if we change our mind.